contracts/reputationMiningCycle/ReputationMiningCycleDataTypes.sol

```diff
@@ -62,12 +62,15 @@ contract ReputationMiningCycleDataTypes {
                                           // Justification tree.
   }
 
+
+  enum disputeStages { CONFIRM_JRH, BINARY_SEARCH_RESPONSE, BINARY_SEARCH_CONFIRM, RESPOND_TO_CHALLENGE, INVALIDATE_HASH, CONFIRM_NEW_HASH }
```

Comment:
Our enum convention is `enum PascalCase { camelCase, camelCase }` -- any reason to depart here?